Reflection/Reflection.php

```diff
@@ -27,7 +27,7 @@ public static function getModifierNames(int $modifiers): array
      * opposed to emitting it. Setting to {@see false} (the default) will do the opposite.
      * @return string|null If the return parameter is set to {@see true}, then the
      * export is returned as a string, otherwise {@see null} is returned.
-     * @deprecated Since PHP 7.4 and removed in 8.0
+     * @deprecated 7.4 and removed in 8.0
```

Comment:
Still..please split to 
```
@deprecated 7.4
@removed 8.0
```
PhpStorm has dedicated support to properly highlight entities with `@removed` tag based on language level

Reply:
As u wish https://github.com/JetBrains/phpstorm-stubs/pull/857/commits/5ecd612d071a18bf6ff435f87de4626d835538af